src/Messages/Attachments/TSAttachmentPointer.h

```diff
@@ -17,11 +17,13 @@ typedef NS_ENUM(NSUInteger, TSAttachmentPointerState) {
  */
 @interface TSAttachmentPointer : TSAttachment
 
+- (instancetype)init NS_UNAVAILABLE;
+
 - (instancetype)initWithServerId:(UInt64)serverId
                              key:(NSData *)key
                           digest:(nullable NSData *)digest
                      contentType:(NSString *)contentType
-                           relay:(NSString *)relay NS_DESIGNATED_INITIALIZER;
```

Comment:
Thanks for addressing our compiler warnings, but could you revert this change? 

It's not pertinent to the rest of this PR, and I'd like to address it differently - the changes I'd like to make will be more concise as a PR rather than a verbose comment here.